recipes/hyperscan/all/patches/fix-cmake.patch

```diff
@@ -0,0 +1,251 @@
+diff --git CMakeLists.txt CMakeLists.txt
+index 8bc6077..60c2ffd 100644
+--- CMakeLists.txt
++++ CMakeLists.txt
+@@ -1,4 +1,4 @@
+-cmake_minimum_required (VERSION 2.8.11)
++cmake_minimum_required (VERSION 3.0.0)
```

Comment:
why is this being changed?

Reply:
Support for 2.8.11 is deprecated and yields a warning (at least with CMake 3.19.3). To avoid the warning I bumped the version, and since version 3 is already very widespread, I decided to bump it to 3.0. As I see it, the changes introduced do not require a newer CMake version. So either don't change it at all and live with the deprecation warning, leave the patch as it is, on figure out what the oldest non-deprecated 2.8 version is and bump the minimum to that. Any suggestions?

Reply:
Warnings are fine, consumer wont see them.

Conan V1.x still supports CMake 2.x, so there's no need to artificially raise the minimum

Reply:
Please revert this since it's not required